### DISCUSSION TITLE
IPAddress5 missing in documentation

### DISCUSSION THREAD

**Discussion by wrightsonm** [arendst/Tasmota#14951] [Q&A] [answered]
### PROBLEM DESCRIPTION
The documentation is missing IPAddress5.  Secondary DNS is now supported since v10.0.0 Norman
https://tasmota.github.io/docs/Commands/#wi-fi

<img width="557" alt="image" src="https://user-images.githubusercontent.com/6232774/155232196-9504373d-edf6-49b4-b9b7-f69f65ca8d45.png">

### REQUESTED INFORMATION
_Make sure your have performed every step and checked the applicable boxes before submitting your issue. Thank you!_

- [x ] Read the [Contributing Guide and Policy](https://github.com/arendst/Tasmota/blob/development/CONTRIBUTING.md) and [the Code of Conduct](https://github.com/arendst/Tasmota/blob/development/CODE_OF_CONDUCT.md)
- [x ] Searched the problem in [issues](https://github.com/arendst/Tasmota/issues)
- [ x] Searched the problem in [discussions](https://github.com/arendst/Tasmota/discussions)
- [x ] Searched the problem in the [docs](https://tasmota.github.io/docs/FAQ)
- [x ] Searched the problem in the [chat](https://discord.gg/Ks2Kzd4)
- [ ] Device used (e.g., Sonoff Basic): _N/A____
- [ ] Tasmota binary firmware version number used: _N/A____
  - [ ] Pre-compiled
  - [ ] Self-compiled
- [ ] Flashing tools used: _____
- [ ] Provide the output of command: `Backlog Template; Module; GPIO 255`:
```lua
  Configuration output here:


```
- [ ] If using rules, provide the output of this command: `Backlog Rule1; Rule2; Rule3`:
```lua
  Rules output here:


```
- [ ] Provide the output of this command: `Status 0`:
```lua
  STATUS 0 output here:


```
- [ ] Set `weblog` to 4 and then, when you experience your issue, provide the output of the Console log:
```lua
  Console output here:


```

### TO REPRODUCE
_Steps to reproduce the behavior:_

### EXPECTED BEHAVIOUR
_A clear and concise description of what you expected to happen._

### SCREENSHOTS
_If applicable, add screenshots to help explain your problem._

### ADDITIONAL CONTEXT
_Add any other context about the problem here._

**(Please, remember to close the issue when the problem has been addressed)**

**Comment by sfromis** [comment, marked as answer]
**You** can be the one adding it. There's an edit link on each page, wiki-style.

**Reply by wrightsonm** [reply]
PR Raised
https://github.com/tasmota/docs/pull/937

**Reply by ascillato2** [reply]
Thanks !!!!
